defs.bzl

```diff
@@ -18,29 +19,46 @@ _shader_stages = [
     'compute',
 ]
 
-def _SpvOutput(srcs):
-    return [src.basename + '.spv' for src in srcs]
-
-def _SpvasmOutput(srcs):
-    return [src.basename + '.spvasm' for src in srcs]
+def _gather_incdirs(files):
+    # Use a dict to avoid duplicates (sets are not allowed in Starlark)
+    incdirs_dict = {}
+    for f in files:
+        path = f.path
+        if "/" in path:
+            parent_dir = path.rsplit("/", 1)[0]
+            incdirs_dict[parent_dir] = True
+    incdirs_list = []
+    for d in sorted(incdirs_dict.keys()):
+        incdirs_list.append("-I" + d)
```

Comment:
I don't think having every source file parent path as an include directory for every file is very expected, no?

Does `glslc` not already find headers relative to the source files?